frontends/web/src/app.tsx

```diff
@@ -199,8 +200,12 @@ class App extends Component<Props, State> {
                             devices={devices}
                             accounts={accounts}
                             deviceIDs={deviceIDs} />
+                        <BuyInfo
+                            path="/buy/info/:code?"
+                            devices={devices}
+                            accounts={accounts} />
                         <Buy
-                            path="/account/:code/buy"
+                            path="/buy/buy/:code"
```

Comment:
I think it would be good to encode the provider here: `/buy/:provider/:code`. So moonpay would be then `/buy/moonpay/:code` and in the future we might have `/buy/bity/:code`, for example.

Then buy/buy.tsx turns into buy/moonpay.tsx and a hypothetical bity would be at buy/bity.tsx.